src/neo/VM/Helper.cs

```diff
@@ -219,6 +220,52 @@ public static byte[] MakeScript(this UInt160 scriptHash, string operation, param
             }
         }
 
+        public static JObject ToJson(this StackItem item)
+        {
+            return ToJson(item, null);
+        }
+
+        private static JObject ToJson(StackItem item, HashSet<StackItem> context)
+        {
+            JObject json = new JObject();
+            json["type"] = item.Type;
+            switch (item)
+            {
+                case Array array:
+                    context ??= new HashSet<StackItem>(ReferenceEqualityComparer.Default);
+                    if (!context.Add(array)) throw new InvalidOperationException();
+                    json["value"] = new JArray(array.Select(p => ToJson(p, context)));
+                    break;
+                case Boolean boolean:
+                    json["value"] = boolean.ToBoolean();
+                    break;
+                case Buffer buffer:
+                    json["value"] = Convert.ToBase64String(buffer.InnerBuffer);
+                    break;
+                case ByteString byteString:
+                    json["value"] = Convert.ToBase64String(byteString.Span);
```

Comment:
I think we don't need base64 string here, just hex string or byte array is enough.  And the json object is for RPC and cli,  not for core to storage, which means it dose not need to be compressed.

Reply:
What the difference between base64 and hex? Neither of them is human readable.

Reply:
base64 is smaller

Reply:
Also it's usual to use base64, please see https://www.newtonsoft.com/json/help/html/SerializationGuide.htm

`Byte[] | String (base 64 encoded)`

Reply:
Got, we have different user habbit. 😂